### PR TITLE
[full-ci] [tests-only] Added tests for `MOVE` request to endpoints with body

### PR DESCRIPTION
**Pull request by grgprarup**
Signed-off-by: Prarup Gurung <grgprarup@gmail.com>

## Description
This PR adds the test for the `MOVE` request to endpoints with body for `oCIS` backend and bug demonstration for `oC10` backend.

## Related Issue

- Part of https://github.com/owncloud/core/issues/40144

## Motivation and Context
webDAV DELETE, COPY or MOVE requests should not send a body in the request.
https://datatracker.ietf.org/doc/html/rfc4918#section-8.4
If a body is sent, then the server is supposed to return HTTP status 415 (Unsupported Media Type).

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by grgprarup** [comment]
The added tests are running for oCIS https://drone.owncloud.com/owncloud/ocis/13920/44/7#:~:text=Scenario%3A%20send%20MOVE%20requests%20to%20webDav%20endpoints%20with,1766
and https://drone.owncloud.com/owncloud/ocis/13920/44/7#:~:text=Scenario%3A%20send%20MOVE%20requests%20to%20another%20user%27s%20webDav%20endpoints%20with,1778

File: tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature [hunk 1]
```diff
@@ -158,3 +158,24 @@ Feature: MOVE file/folder
...
+      | /remote.php/dav/files/%username%/textfile0.txt     |
+      | /remote.php/webdav/PARENT            |
+      | /remote.php/dav/files/%username%/PARENT            |
+      | /remote.php/webdav/PARENT/parent.txt |
+      | /remote.php/dav/files/%username%/PARENT/parent.txt |
+    Then the HTTP status code of responses on all endpoints should be "415"
+
+  @skipOnOcV10 @personalSpace
+  Scenario: send MOVE requests to another user's webDav endpoints with body as normal user using the spaces WebDAV API
+    When user "Alice" requests these endpoints with "MOVE" including body "doesnotmatter" about user "Alice"
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature:176]:
```suggestion
    When user "Brian" requests these endpoints with "MOVE" including body "doesnotmatter" about user "Alice"
```
another user's webdav endpoint is to be checked

File: tests/acceptance/features/apiAuthWebDav/webDavMOVEAuthOC10Issue40144.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      | Brian    |
+    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
+    And user "Alice" has created folder "/PARENT"
+    And user "Alice" has created folder "/FOLDER"
+    And user "Alice" has uploaded file with content "some data" to "/PARENT/parent.txt"
+
+
+  Scenario: send MOVE requests to webDav endpoints with body as normal user
+    When user "Alice" requests these endpoints with "MOVE" including body "doesnotmatter" about user "Alice"
+      | endpoint                                           |
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiAuthWebDav/webDavMOVEAuthOC10Issue40144.feature:17]: format the table.

File: tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature (hunk 1 above)
Comment (grgprarup) [reply tests/acceptance/features/apiAuthWebDav/webDavMOVEAuth.feature:176]: @kiranparajuli589 Actually this should be the same as https://github.com/owncloud/core/pull/40251/files#:~:text=%40skipOnOcV10-,Scenario%3A%20send%20MOVE%20requests%20to%20webDav%20endpoints%20with%20body%20as%20normal%20user,-When%20user%20%22Alice
Forgot to change here. Fixed it.

File: tests/acceptance/features/apiAuthWebDav/webDavMOVEAuthOC10Issue40144.feature (hunk 2 above)
Comment (grgprarup) [reply tests/acceptance/features/apiAuthWebDav/webDavMOVEAuthOC10Issue40144.feature:17]: Done. Thank you.

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Review by SagarGi — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**